src/providers/details/tabs/Hosts/components/utils/validators.ts

```diff
@@ -0,0 +1,9 @@
+import { validateNoSpaces } from 'src/modules/Providers/utils/validators/common';
+
+export const validateUsername = (username: string) => {
+  return validateNoSpaces(username);
+};
+
+export const validatePassword = (password: string) => {
+  return validateNoSpaces(password);
+};
```

Comment:
What's the point of those functions?

Reply:
Because I thought there would be more validations.
Removed.